test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -26,11 +27,11 @@ public void ConstructorDoesntThrow()
         {
             Assert.NotNull(new TextLoader<Input>("fakeFile.txt"));
             Assert.NotNull(new TextLoader<Input>("fakeFile.txt", useHeader: true));
-            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", separator: "tab"));
-            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", useHeader: false, separator: "tab"));
-            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", useHeader: false, separator: "tab", false, false));
-            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", useHeader: false, separator: "tab", supportSparse: false));
-            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", useHeader: false, separator: "tab", allowQuotedStrings: false));
+            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", delimeter: ','));
+            Assert.NotNull(new TextLoader<Input>("fakeFile.txt", useHeader: false, delimeter: ','));
```

Comment:
>',' [](start = 94, length = 3)

I'm not sure it matters given the nature of this test, but was there any particular reason to shift from `tab` to `,`?

Given how you've made `\t` an exceptional case in the constructor, might want to do both.

Reply:
This was on an accident. Reverted back.

---
In reply to: [188011098](https://github.com/dotnet/machinelearning/pull/142#discussion_r188011098) [](ancestors = 188011098)